docs/deploying_cloud_provider_vsphere_with_rbac.md

```diff
@@ -1,13 +1,13 @@
 # Deploying `cloud-provider-vsphere` with RBAC
 
-This document is designed to quickly get you up and running with the `cloud-provider-vsphere` external Cloud Controller Manager (CCM).
+This document is designed to quickly get you setting up and running with the `cloud-provider-vsphere` external Cloud Controller Manager (CCM).
```

Comment:
disclaimer: I'm a non-native speaker

Is `setting` really needed in this sentence? I've always seen the form

> get you up and running

Reply:
Hi, Frapposelli, I've checked and confirmed that you are right. 
Thanks!